contrib/babelfishpg_common/sql/upgrades/babelfishpg_common--2.3.0--2.4.0.sql

```diff
@@ -0,0 +1,54 @@
+-- complain if script is sourced in psql, rather than via ALTER EXTENSION
```

Comment:
why do we need to update this upgrade script if this change is only meant for 3_X_DEV branch?

Reply:
Fixed in latest commit